### PR TITLE
test GetApplicationLog

### PR DESCRIPTION
**Pull request by Hecate2**
# Description

Adding tests

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/Neo.Plugins.ApplicationLogs.Tests/UT_LogReader.cs [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+        private readonly NeoSystemFixture _neoSystemFixture;
+
+        public UT_LogReader(NeoSystemFixture neoSystemFixture)
+        {
+            _neoSystemFixture = neoSystemFixture;
+        }
+
+        [Fact]
+        async public void Test_GetApplicationLog()
```
Comment (cschuchardt88) [inline tests/Neo.Plugins.ApplicationLogs.Tests/UT_LogReader.cs:137]:
```suggestion
        public async Task Test_GetApplicationLog()
```

consistency

File: tests/Neo.Plugins.ApplicationLogs.Tests/UT_LogReader.cs [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            Assert.Equal(notifications[0]["eventname"].AsString(), "Transfer");
+            Assert.Equal(notifications[0]["contract"].AsString(), NeoToken.NEO.Hash.ToString());
+            Assert.Equal(notifications[0]["state"]["value"][2]["value"], "1");
+            Assert.Equal(notifications[1]["eventname"].AsString(), "Transfer");
+            Assert.Equal(notifications[1]["contract"].AsString(), GasToken.GAS.Hash.ToString());
+            Assert.Equal(notifications[1]["state"]["value"][2]["value"], "50000000");
+        }
+
+        [Fact]
+        async public void Test_Commands()
```
Comment (cschuchardt88) [inline tests/Neo.Plugins.ApplicationLogs.Tests/UT_LogReader.cs:178]:
```suggestion
        public async Task Test_Commands()
```

**Review by cschuchardt88 — COMMENTED** [review]
If you want to write better tests you can use `NeoStore` Just below set to `internal` variable.

https://github.com/neo-project/neo/blob/9ce9c6fe1cdce05ef20fff0d6b3ae4b25811054f/src/Plugins/ApplicationLogs/LogReader.cs#L33

Than you can use these functions

https://github.com/neo-project/neo/blob/9ce9c6fe1cdce05ef20fff0d6b3ae4b25811054f/src/Plugins/ApplicationLogs/Store/NeoStore.cs#L72

https://github.com/neo-project/neo/blob/9ce9c6fe1cdce05ef20fff0d6b3ae4b25811054f/src/Plugins/ApplicationLogs/Store/NeoStore.cs#L116

https://github.com/neo-project/neo/blob/9ce9c6fe1cdce05ef20fff0d6b3ae4b25811054f/src/Plugins/ApplicationLogs/Store/NeoStore.cs#L184

**Review by cschuchardt88 — APPROVED** [review]
Nice work!

We try to use `var` type everywhere.

**Review by shargon — APPROVED** [review]

**Review by AnnaShaleva — APPROVED** [review]

**Review by vncoelho — COMMENTED** [review]
Such a good test case!
Congratulations for this amazing job @Hecate2

**Pull request merged**